snippets/csharp.json

```diff
@@ -302,6 +302,13 @@
         ],
         "description": "Namespace"
     },
+    "ShortNamespace": {
+        "prefix": "snamespace",
```

Comment:
I think we should go with either "fsnamespace" or "fsn". What do you think?

Reply:
I think, it was great idea. I change it